test/AspNet.Security.OAuth.Providers.Tests/OAuthTests`1.cs

```diff
@@ -41,12 +41,14 @@ protected OAuthTests()
                 .ThrowsOnMissingRegistration()
                 .RegisterBundle(Path.Combine(BundleName, "bundle.json"));
 
+#pragma warning disable DF0022 // Marks indisposed objects assinged to a property, originated in an object creation.
```

Comment:
What tooling raises this warning?

Reply:
https://github.com/BADF00D/DisposableFixer

Reply:
Given we don't use this analyzer in the repo source code and this is just something specific to your IDE setup, I'd rather it be removed.

Reply:
But as long as you treat warnings as errors we can't build the project without that.

Reply:
What about adding this analyzer?

Reply:
I'd prefer to just fix the problem it's identifying (even though it's not an issue in practice), rather than add an extra analyser just because you happen to have it installed in your personal Visual Studio installation.